packages/playground-examples/copy/pt/Playground/Tooling/Sharable URLs.ts

```diff
@@ -0,0 +1,30 @@
+//// { order: 1 }
+
+// Com o novo Playground, os URLs tem muito mais informações
+// sobre suas configurações. Isso faz com que reproduções
+// sejam precisas, e que você não tenha que falar para alguém
+// usar uma determinada flag de compilador para replicar a
+// sua configuração.
+
+// A URL contem:
+//
+// - Se você esta em um exemplo
+// - O código-fonte compactado para o seu TypeScript/JavaScript
+// - Retrocompatilibilidade com URLs antigas não compactadas
+// - Em qual idioma você está trabalhando
+// - A diferença entre suas configurações de compilação e 
+//   a configuração padrão
+//   Por exemplo:new-compiler-defaults
```

Comment:
```suggestion
//   Por exemplo: example:new-compiler-defaults
```
`example:*` are links and must not be translated directly